src/app/core/utils/http-error/pgid-change.error-handler.ts

```diff
@@ -0,0 +1,11 @@
+import { SpecialHttpErrorHandler } from 'ish-core/interceptors/icm-error-mapper.interceptor';
+
+// reload page when the pgid of the user has changed
+export const pgidChangeErrorHandler: SpecialHttpErrorHandler = {
+  test: error =>
+    error.error === 'Bad Request (The provided matrix parameter "pgid" does not match the authenticated user.)',
+  map: () => {
+    window.location.reload();
```

Comment:
If a HttpInterceptor should be introduced, than it makes sense to reload the page only on browser side.

Reply:
you won't get this error for SSR requests